assets/component-slideshow.css

```diff
@@ -0,0 +1,271 @@
+
+@media screen and (max-width: 749px) {
+  .slideshow--placeholder.banner--mobile-bottom.banner--adapt_image .slideshow__media,
+  .slideshow--placeholder.banner--adapt_image:not(.banner--mobile-bottom) {
+    height: 28rem;
+  }
+}
+
+@media screen and (min-width: 750px) {
+  .slideshow--placeholder.banner--adapt_image {
+    height: 56rem;
+  }
+}
+
+.slideshow__text.banner__box {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  max-width: 54.5rem;
+}
+
+@media screen and (max-width: 749px) {
+  .banner--mobile-bottom .slideshow__text.banner__box {
+    max-width: 100%;
+  }
+
+  .banner--mobile-bottom .slideshow__text-wrapper {
+    flex-grow: 1;
+  }
+
+  .banner--mobile-bottom .slideshow__text.banner__box {
+    height: 100%;
+  }
+
+  .banner--mobile-bottom .slideshow__text .button {
+    flex-grow: 0;
+  }
+
+}
+
+.banner:not(.banner--mobile-bottom) .slideshow__text-wrapper {
+  height: 100%;
+}
+
+@media screen and (min-width: 750px) {
+  .slideshow__text-wrapper {
+    height: 100%;
+    padding: 5rem;
+  }
+
+  .slideshow__text-wrapper.slideshow__text-wrapper--top-left {
+    align-items: flex-start;
+    justify-content: flex-start;
+  }
+
+  .slideshow__text-wrapper.slideshow__text-wrapper--top-center {
+    align-items: flex-start;
+    justify-content: center;
+  }
+
+  .slideshow__text-wrapper.slideshow__text-wrapper--top-right {
+    align-items: flex-start;
+    justify-content: flex-end;
+  }
+
+  .slideshow__text-wrapper.slideshow__text-wrapper--middle-left {
+    align-items: center;
+    justify-content: flex-start;
+  }
+
+  .slideshow__text-wrapper.slideshow__text-wrapper--middle-center {
+    align-items: center;
+    justify-content: center;
+  }
+
+  .slideshow__text-wrapper.slideshow__text-wrapper--middle-right {
+    align-items: center;
+    justify-content: flex-end;
+  }
+
+  .slideshow__text-wrapper.slideshow__text-wrapper--bottom-left {
+    align-items: flex-end;
+    justify-content: flex-start;
+  }
+
+  .slideshow__text-wrapper.slideshow__text-wrapper--bottom-center {
+    align-items: flex-end;
+    justify-content: center;
+  }
+
+  .slideshow__text-wrapper.slideshow__text-wrapper--bottom-right {
+    align-items: flex-end;
+    justify-content: flex-end;
+  }
+}
+
+.slideshow__controls {
+  border: 0.1rem solid rgba(var(--color-foreground), 0.08);
+  background-color: rgba(var(--color-background), 0.95);
+}
+
+@media screen and (max-width: 749px) {
+  .slideshow:not(.banner--mobile-bottom) + .slideshow__controls {
+    background-color: rgb(var(--color-base-background-1));
+    color: rgba(var(--color-base-text), 0.75);
+  }
+
+  .slideshow:not(.banner--mobile-bottom) + .slideshow__controls .slider-button,
+  .slideshow:not(.banner--mobile-bottom) + .slideshow__controls .slider-counter__link--active.slider-counter__link--numbers,
+  .slideshow:not(.banner--mobile-bottom) + .slideshow__controls .slider-counter__link--numbers {
+    color: rgba(var(--color-base-text), 0.75);
+  }
```

Comment:
I need it because they're child elements of `.slideshow`'s sibling. I don't think I have another way to select it 🤔 

Unless I add a class on the `slider-component` which would help reduce the amount of selectors. Ill look into it